addons/html_editor/static/src/core/dom_plugin.js

```diff
@@ -627,9 +627,6 @@ export class DomPlugin extends Plugin {
                 if (extraClass) {
                     newEl.classList.add(extraClass);
                 }
-                if (block.nodeName === "LI") {
-                    this.delegateTo("set_tag_overrides", block, newEl);
-                }
```

Comment:
Why do you remove this without adapting it to something else? Is it maybe because there are no longer any resource with that name?

Reply:
Yes, it's in the commit message :smile: 
> As there's no "set_tag_overrides" resource currently registered in the codebase,
the usage of `delegateTo` with this resource was simply removed.